src/App.css

```diff
@@ -5,7 +5,8 @@
 |*************************************************************************;
 */
 
-@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap');
```

Comment:
Good font and happy to see this now, but let's see that design direction before we commit

Reply:
Will create design document before any more styling.